### PR TITLE
sql-statements: update "admin" related statements

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)
Update "admin" related statements.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by zimulala** [comment]
@yikeke I update `media/sqlgram/AdminStmt.png`, but the sqlgram of `AdminStmt` doesn't update when displaying. Do you know why?

File: sql-statements/sql-statement-admin.md [hunk 1]
```diff
@@ -51,13 +61,113 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
...
+
+此语句可以将出现超过 1 次的 select 语句生成 SQL Plan 的绑定。
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN EVOLVE bindings;
+```
+
+开启自动绑定功能后，每隔 `bind-info-lease`（默认值为 `3s`）触发一次 SQL Plan 绑定信息的演进。此语句用于主动触发此演进，SQL Plan 绑定详情可参考：[执行计划绑定](execution-plan-binding.md)。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin.md:160]:
```suggestion
开启自动绑定功能后，每隔 `bind-info-lease`（默认值为 `3s`）触发一次 SQL Plan 绑定信息的演进。此语句用于主动触发此演进，SQL Plan 绑定详情可参考：[执行计划绑定](/execution-plan-binding.md)。
```

**Comment by yikeke** [comment]
> @yikeke I update `media/sqlgram/AdminStmt.png`, but the sqlgram of `AdminStmt` doesn't update when displaying. Do you know why?

You mean the `media/sqlgram/AdminStmt.png` is not actually updated?

**Comment by zimulala** [comment]
@yikeke 
Yes.

**Comment by yikeke** [comment]
> @yikeke
> Yes.

That‘s strange. Can you send that image to me? I'll upload it to this PR to check what's wrong. @zimulala

**Comment by zimulala** [comment]
@yikeke Done.

File: sql-statements/sql-statement-admin.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ aliases: ['/docs-cn/dev/reference/sql/statements/admin/']
 
 # ADMIN
 
-`ADMIN` 语句是 TiDB 扩展语法，用于查看 TiDB 自身的状态，并对 TiDB 中的表数据进行校验。示例如下。
+`ADMIN` 语句是 TiDB 扩展语法，用于查看 TiDB 自身的状态，并对 TiDB 中的表数据进行校验。
+
+## ADMIN 与 DDL 相关的扩展语句
+
+### `admin show DDL` 命令
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin.md:13]: `命令` --> `语句`？

File: sql-statements/sql-statement-admin.md [hunk 3]
```diff
@@ -51,13 +61,113 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
...
+### `admin check` 命令
+
 {{< copyable "sql" >}}
 
 ```sql
 ADMIN CHECK TABLE tbl_name [, tbl_name] ...;
 ```
 
-`ADMIN CHECK TABLE tbl_name [, tbl_name] ...` 用于对表 `tbl_name` 中的所有数据和对应索引进行一致性校验。若通过校验，则返回空的查询结果；否则返回数据不一致的错误信息。
+此语句用于对表 `tbl_name` 中的所有数据和对应索引进行一致性校验。若通过校验，则返回空的查询结果；否则返回数据不一致的错误信息。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin.md:72]: 用`以上`会不会更明确一些？
```suggestion
以上语句用于对表 `tbl_name` 中的所有数据和对应索引进行一致性校验。若通过校验，则返回空的查询结果；否则返回数据不一致的错误信息。
```

File: sql-statements/sql-statement-admin.md [hunk 4]
```diff
@@ -51,13 +61,113 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
...
+
+此语句用于对 `tbl_name` 表中 `idx_name` 索引对应列数据和索引数据进行一致性校验。若通过校验，则返回空的查询结果；否则返回数据不一致的错误信息。
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CHECK INDEX tbl_name idx_name (lower_val, upper_val) [, (lower_val, upper_val)] ...;
+```
+
+此语句用于对 `tbl_name` 表中 `idx_name` 索引对应列数据和索引数据进行一致性校验, 并且指定了需要检查的数据范围。若通过校验，则返回空的查询结果；否则返回数据不一致的错误信息。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin.md:88]:
```suggestion
此语句用于对 `tbl_name` 表中 `idx_name` 索引对应列数据和索引数据进行一致性校验，并且指定了需要检查的数据范围。若通过校验，则返回空的查询结果；否则返回数据不一致的错误信息。
```

File: sql-statements/sql-statement-admin.md [hunk 5]
```diff
@@ -51,13 +61,113 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
...
+
+### `admin checksum` 命令
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CHECKSUM TABLE tbl_name [, tbl_name] ...;
+```
+
+此语句会获取 `tbl_name` 的 64 位的 checksum 值，此值是通过计算了表中所有的键值对（包括行数据和索引数据）的 CRC64 获得的。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin.md:98]:
```suggestion
以上语句会获取 `tbl_name` 的 64 位的 checksum 值，该值可通过计算表中所有的键值对（包括行数据和索引数据）的 CRC64 获得。
```

File: sql-statements/sql-statement-admin.md [hunk 6]
```diff
@@ -51,13 +61,113 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
...
+
+此语句用于持久化 SQL Plan 绑定的信息。
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN CAPTURE bindings;
+```
+
+此语句可以将出现超过 1 次的 select 语句生成 SQL Plan 的绑定。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin.md:152]:
```suggestion
此语句可以将出现超过一次的 `select` 语句生成 SQL Plan 的绑定。
```

File: sql-statements/sql-statement-admin.md [hunk 7]
```diff
@@ -51,13 +61,113 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...;
...
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN EVOLVE bindings;
+```
+
+开启自动绑定功能后，每隔 `bind-info-lease`（默认值为 `3s`）触发一次 SQL Plan 绑定信息的演进。此语句用于主动触发此演进，SQL Plan 绑定详情可参考：[执行计划绑定](/execution-plan-binding.md)。
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin.md:160]:
```suggestion
开启自动绑定功能后，每隔 `bind-info-lease`（默认值为 `3s`）触发一次 SQL Plan 绑定信息的演进。以上语句用于主动触发此演进，SQL Plan 绑定详情可参考：[执行计划绑定](/execution-plan-binding.md)。
```

**Comment by zimulala** [comment]
@TomShawn 
Done. PTAL

**Pull request merged**